src/SFA.DAS.EmployerIncentives.Api.UnitTests/Controllers/PassThroughControllerTests.cs

```diff
@@ -0,0 +1,109 @@
+    {
+        [Test]
+        public async Task When_AddLegalEntity_is_requested_Then_EmployerIncentivesPassThroughService_is_called()
+        {
+            var f = new TestsFixture().SetupEmployerIncentivesServiceForAddLegalEntityCall();
+
+            await f.Sut.AddLegalEntity(f.AccountId, f.LegalEntityRequest);
+
+            f.EmployerIncentivesPassThroughServiceMock.Verify(x => x.AddLegalEntity(f.AccountId, f.LegalEntityRequest));
+        }
+
+        [Test]
+        public async Task When_AddLegalEntity_is_requested_Then_EmployerIncentivesPassThroughService_returns_Content_and_Statuscode()
+        {
+            var f = new TestsFixture().SetupEmployerIncentivesServiceForAddLegalEntityCall();
+
+            var result = await f.Sut.AddLegalEntity(f.AccountId, f.LegalEntityRequest);
+
+            f.VerifyInnerApiStatusCodeAndContentIsReturnFromOuterApi(result);
+        }
+
+        [Test]
+        public async Task When_RemoveLegalEntity_is_requested_Then_EmployerIncentivesPassThroughService_is_called()
+        {
+            var f = new TestsFixture().SetupEmployerIncentivesServiceForRemoveLegalEntityCall();
+
+            await f.Sut.RemoveLegalEntity(f.AccountId, f.AccountLegalEntityId);
+
+            f.EmployerIncentivesPassThroughServiceMock.Verify(x => x.RemoveLegalEntity(f.AccountId, f.AccountLegalEntityId));
+        }
+
+        [Test]
+        public async Task When_RemoveLegalEntity_is_requested_Then_EmployerIncentivesPassThroughService_returns_Content_and_Statuscode()
+        {
+            var f = new TestsFixture().SetupEmployerIncentivesServiceForRemoveLegalEntityCall();
+
+            var result = await f.Sut.RemoveLegalEntity(f.AccountId, f.AccountLegalEntityId);
+
+            f.VerifyInnerApiStatusCodeAndContentIsReturnFromOuterApi(result);
+        }
+
+        private class TestsFixture
+        {
+            public Mock<IEmployerIncentivesPassThroughService> EmployerIncentivesPassThroughServiceMock;
+            public PassThroughController Sut;
+            public Fixture Fixture;
+            public long AccountId;
+            public long AccountLegalEntityId;
+            public LegalEntityRequest LegalEntityRequest;
+            public InnerApiResponse InnerApiResponse;
+
+            public TestsFixture()
+            {
+                Fixture = new Fixture();
+                EmployerIncentivesPassThroughServiceMock = new Mock<IEmployerIncentivesPassThroughService>();
+
+                Sut = new PassThroughController(EmployerIncentivesPassThroughServiceMock.Object);
+
+                AccountId = Fixture.Create<long>();
+                AccountLegalEntityId = Fixture.Create<long>();
+                LegalEntityRequest = Fixture.Create<LegalEntityRequest>();
+                InnerApiResponse = Fixture.Build<InnerApiResponse>().Without(x=>x.Json).Create();
+            }
+
+            public TestsFixture SetupEmployerIncentivesServiceForAddLegalEntityCall()
```

Comment:
None of these seem very specific to the actual request, we shouldn't be doing It.IsAny for anything other than checking that something is never called

Reply:
Perhaps these can be tested fully via the Acceptance Tests